ui-tests/tests/notebooks/widgets.ipynb

```diff
@@ -6,15 +6,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import ipywidgets as widgets"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    "import ipywidgets as widgets\n",
```

Comment:
Moving the import to the cell below, since Galata would otherwise block on trying to fetch the cell output with `galata.notebook.getCellOutput()` when there is no output.